models/OrganisationRestApiModel.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const HmpoModel = require('hmpo-model');
+const { iamApi } = require('../config');
+const baseUrl = `${iamApi.protocol}://${iamApi.host}:${iamApi.port}`;
```

Comment:
Is there a way we can abstract this somewhere? I imagine we'll need this in quite a few places

Reply:
Done